lnrpc/walletrpc/walletkit.proto

```diff
@@ -108,6 +124,35 @@ service WalletKit {
         returns (LabelTransactionResponse);
 }
 
+message LeaseOutputRequest {
+    /*
+    An ID of 32 random bytes that must be unique for each distinct application
+    using this RPC which will be used to bound the output lease to.
+    */
+    bytes id = 1;
+
+    // The identifying outpoint of the output being leased.
+    lnrpc.OutPoint outpoint = 2;
```

Comment:
We use "outpoint" here, but the calls all use "output". Thinking we should make it consistent here and use "outpoint" everywhere on the RPC level.

Reply:
Well you really are leasing the output itself, with outpoint being how you refer to it.